### ISSUE TITLE
Misleading description of packages module

### ISSUE THREAD

**Issue by kkofler** [calamares/calamares#899]
The UI shows the `packages` module as
`Calls routine with detected package manager to install locale packages`
which is a truncated (from https://github.com/calamares/calamares/blob/master/src/modules/packages/main.py#L426) and misleading description. And even the full 2 lines
```
    Calls routine with detected package manager to install locale packages
    or remove drivers not needed on the installed system.
```
are too specific and not true in some cases, in particular if the `netinstall` module is used.

**Comment by kkofler** [comment]
I'd suggest something like `Installing and/or removing packages`.

**Comment by kkofler** [comment]
This should already be fixed (see commit 517dbfab0), sorry for the noise.
